### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by erezrokah**
Will close this if we get renovate approval for the repo.
For now it updates all deps to latest non major versions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,16 +30,16 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@netlify/build": "^3.1.10",
-    "ava": "^3.11.1",
+    "@netlify/build": "^3.3.5",
```
Comment (ehmicky) [inline package.json:33]: This should be safe to upgrade to latest major.

File: package.json [hunk 2]
```diff
@@ -48,13 +48,13 @@
     "fuzzy": "^0.1.3",
     "git-remote-origin-url": "^3.1.0",
     "inquirer": "^7.3.3",
-    "inquirer-autocomplete-prompt": "^1.0.2",
-    "netlify-cli": "^2.59.0",
+    "inquirer-autocomplete-prompt": "^1.2.0",
+    "netlify-cli": "^2.65.0",
```
Comment (ehmicky) [inline package.json:52]: Same here

**Comment by erezrokah** [comment]
https://github.com/release-it/release-it/blob/master/CHANGELOG.md#v14
https://github.com/sindresorhus/del/releases/tag/v6.0.0
seem safe.

https://github.com/yargs/yargs/releases/tag/v16.0.0 not so much without testing

File: package.json (hunk 2 above)
Comment (erezrokah) [reply package.json:52]: Which one `netlify-cli` latest is `2.65.0`

Comment (ehmicky) [reply package.json:52]: Oh ok, this is good then :+1:

**Review by ehmicky — APPROVED** [review]

**Pull request merged**